### PR TITLE
fix: added image with responsive screen

### PR DESCRIPTION
**Pull request by poojakelkar**
-added responsive image

### CODE REVIEW

**Review by chanchal-panpaliya — COMMENTED** [review]
The code looks good and the images is responsive, but the class name is not consistent. For example, you are assigning a class name -  "img-fullscreen" , for the child "fullscreen". Visit [https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/ for more information.](url)

File: components/image/index.html [hunk 1]
```diff
@@ -21,5 +21,12 @@
                 src="https://images.unsplash.com/photo-1447752875215-b2761acb3c5d?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80"
             />
         </div>
+
+        <div class="img-fullscreen">
+            <img
+                class="fullscreen"
+                src="https://images.unsplash.com/photo-1619441207978-3d326c46e2c9?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1169&q=80"
+            />
```
Comment (shrushti2000) [inline components/image/index.html]: The alt attribute is missing.

**Comment by shrushti2000** [comment]
Just look at the naming conventions once. Else I think it is good to be merged.

File: components/image/index.html (hunk 1 above)
Comment (poojakelkar) [reply components/image/index.html]: yeah, added now

**Comment by poojakelkar** [comment]
@chanchal-panpaliya yeah I have changed it. thank you

**Pull request merged**